### PR TITLE
Make it Work

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Resolves https://github.com/ember-codemods/ember-no-implicit-this-codemod/issues/1
Blocked by codemod-cli not being able to handle `hbs` as a file extension

Apparently I can't make PR a draft after creating it :(

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    "debug:codemod": "node --inspect-brk node_modules/jscodeshift/bin/jscodeshift.js -t ./transforms/no-implicit-this/index.js --run-in-band --extensions js,ts,hbs "
+  },
+  "bin": {
+    "ember-no-implicit-this-codemod": "./bin/cli.js"
+  },
+  "keywords": [
+    "codemod-cli"
+  ],
+  "dependencies": {
+    "babel-plugin-htmlbars-inline-precompile": "^1.0.0",
```
Comment (rwjblue) [inline package.json]: Should be a dev dep

File: package.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    "codemod-cli": "^1.0.0",
+    "ember-template-compiler": "^1.9.0-alpha",
```
Comment (rwjblue) [inline package.json]: Shouldn’t use this

File: transforms/no-implicit-this/__testfixtures__/component-invocation.input.hbs [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{{some-component}}
```
[the remark this answers is not in the document]
Comment (NullVoxPopuli) [reply transforms/no-implicit-this/__testfixtures__/component-invocation.input.hbs]: can do :+1:

File: lib/gather-telemetry.js [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+       * @returns {String[]} data.observedProperties - list of observed properties
+       * @returns {Object} data.observerProperties - list of observer properties
+       * @returns {Object} data.offProperties - list of observer properties
+       * @returns {String[]} data.overriddenActions - list of overridden actions
+       * @returns {String[]} data.overriddenProperties - list of overridden properties
+       * @returns {String[]} data.ownProperties - list of object's own properties
+       * @returns {String} data.type - type of ember object
+       * @returns {Object} data.unobservedProperties - list of unobserved properties
+       */
+      function parseMeta(meta = {}) {
```
[the remark this answers is not in the document]
Comment (NullVoxPopuli) [reply lib/gather-telemetry.js]: I removed some stuff that the native class codemod was doing :man_shrugging: 
should getting telemetry for codemods be its own package though? just have one package to get _everything_?

[the remark this answers is not in the document]
Comment (rwjblue) [reply lib/gather-telemetry.js]: The issue is that "everything" is super nebulous, I think for now its better to directly depend on puppeteer and do what is needed for each. Once we have a better idea of the infrastructure, we can abstract to a shared package (kinda like what I did with codemod-cli to begin with.

**Comment by rwjblue** [comment]
DAMMIT, I meant to merge not close 😩

**Pull request merged**